### PR TITLE
Display Popup with selected item's comments

### PR DESCRIPTION
**Pull request by jaflih**
### Display Pop up with selected item's comments

- Added show comment for a specific pokemon

### CODE REVIEW

File: src/modules/api.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 export default class Api {
   POKEMON_URL = 'https://pokeapi.co/api/v2/pokemon';
```
Comment (matovu-farid) [inline src/modules/api.js]: The api breaks on newer browser. Please check out the branch called api-fix to view the fix. You can then merge and push again
To understand why it breaks, check out this issue
- [click me](https://github.com/PokeAPI/pokeapi/issues/289)

**Review by matovu-farid — CHANGES_REQUESTED** [review]
Great job Jihanne 👏 , when the fix is applied, The comments can be viewed. 
However, there are a few issues to address. 
- In order to help you out, I have included a branch called api-fix with the fix to the issue 
- The develop branch changes, please merge the develop branch into this branch

File: src/modules/api.js (hunk 1 above)
Comment (jaflih) [reply src/modules/api.js]: @matovu-farid thank's for the review. There is no difference between this branch and the api-fix branch

Capture d’écran 2022-02-02 à 06 34 17

**Review by matovu-farid — APPROVED** [review]
There is nothing left to say, it's time to merge

**Pull request merged**